src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -23,4 +23,39 @@ public int divide(int a, int b) throws IllegalArgumentException {
         }
```

Comment:
1. **Error Handling:** The `divide` method appropriately checks for division by zero and throws an `IllegalArgumentException`. This is good practice for handling exceptional cases.